File: Beecrowd/1046 Tempo de Jogo/main.py
x, y = [int(_) for _ in input().split()]

if x == y:
    print("O JOGO DUROU 24 HORA(S)")

if x > y:
    t = 24 - x + y
    print(f"O JOGO DUROU {t} HORA(S)")

if x < y:
    t = y - x
    print(f"O JOGO DUROU {t} HORA(S)")

File: Beecrowd/1046 Tempo de Jogo/main.c
#include <stdio.h>
 
int main() {
 
	int X, Y, T;
	
	scanf("%d %d", &X, &Y);
	
	if(X == Y)
		printf("O JOGO DUROU 24 HORA(S)\n");
	
	if(X > Y){
		T = 24 - X + Y;
		printf("O JOGO DUROU %d HORA(S)\n", T);
	}
	
	if(X < Y){
		T = Y - X;
		printf("O JOGO DUROU %d HORA(S)\n", T);
	}
 
    return 0;
}